javaws-wrapper/src/main/java/com/teamscale/jacoco/javaws/WindowsInstallation.kt

```diff
@@ -92,7 +90,7 @@ constructor(workingDirectory: Path) {
         }
 
         try {
-            FileSystemUtils.writeFileUTF8(backupPaths.ftypeMapping.toFile(), readCurrentJnlpFtype())
+            backupPaths.ftypeMapping.toFile().writeText(readCurrentJnlpFtype())
```

Comment:
does writetext actually throw IO Exceptions? the documentation does not say. please check and adjust catch clause if necessary

Reply:
It calls `FileOutputStream` internally, which may throw a `FileNotFoundException` and `write` which throws an `IOException`.
https://discuss.kotlinlang.org/t/in-future-could-kotlin-have-checked-exception/1579/15